### ISSUE TITLE
Unable to resolve asset paths when using nwayo-bg-image-styles mixin

### ISSUE THREAD

**Issue by rbrlortie** [absolunet/nwayo#48]
There seem to be issues with the directory separators on Windows 10. 

It attempts to retrieve the assets using this path `'C:xampphtdocsmagentosrcskin�rontendrb‘�ixtures�uild/images/common/logo.jpg'` which is obliviously broken.

site.yaml
```
output:
  konstan: ../src/skin/frontend/rb/2018/fixtures/build
  build:   ../src/skin/frontend/rb/2018/fixtures/build
  url:     /build
```

_header.scss
```
@include nwayo-bg-image-styles('logo.jpg', 'common');
```

console
```
[16:29:29] Starting task styles-compile
Error in plugin "sass"
Message:
    bower_components\nwayo-toolbox\toolbox\styles\_functions-core.scss
Error: File 'C:xampphtdocsmagentosrcskinrontendrb‘ixturesuild/images/common/logo.jpg' not found
        @return '?v=' + nwayodart-checksum($file);
                        ^^^^^^^^^^^^^^^^^^^^^^^^^
  bower_components\nwayo-toolbox\toolbox\styles\_functions-core.scss 59:18  nwayo-cacheinvalidator-querystring()
  bower_components\nwayo-toolbox\toolbox\styles\_mixins-core.scss 56:20     nwayo-bg-image-styles()
  components\site\styles\_header.scss 125:2                                 @import
  components\site\styles\site.scss 8:9                                      @import
  main.undefined 10:9 
```

file is present on filesystem:
![image](https://user-images.githubusercontent.com/23345449/48513958-3da10280-e82b-11e8-88f9-b6d1cb7b1e7e.png)

**Comment by rbrlortie** [comment]
Or is `nwayo-bg-image-styles` meant to be used only by nwayo's core? In that case, is there a mixin that outputs asset paths based on the bundle name?

**Comment by jblandry** [comment]
The function is trying to find an image 

`build/images/common/logo.jpg`

but your image is 

`build/images/site/logo.png`

**Comment by rbrlortie** [comment]
Apologies. The screenshot and snippets from in the initial issue are from two different attempts.

Here is an updated image of what's happening:

![image](https://user-images.githubusercontent.com/23345449/49405526-adb2f200-f720-11e8-9319-857abd52c277.png)


along with my magento.yaml file:

```
#-- Output paths (relative to nwayo folder) and output url
output:
  konstan: ../src/skin/frontend/rb/2018/fixtures/build
  build:   ../src/skin/frontend/rb/2018/fixtures/build
  url:     /build
```

Seems like `f` becomes `�` and `2018` becomes `‘`.

**Comment by jblandry** [comment]
Can't reproduce the encoding errors on Windows setup, you should investigate how the paths get manipulated in the https://github.com/absolunet/nwayo/blob/release/next/workflow/helpers/paths.js file.

**Comment by jblandry** [comment]
Corrected via 997c1c4e16cbedd9603ac45743ed2e7790e9608e

**Comment by rbrlortie** [comment]
Confirmed
